### ISSUE TITLE
Feature request: Please add a mode for bookmark only.

### ISSUE THREAD

**Issue by cocoonkid** [samhh/bukubrow-webext#151]
As a user I would like to run bukobrow in a bookmark only mode with two options.

Onclick either save page directly to bookmarks OR open add bookmark options ONLY.

Also accepting Enter as confirmation to save wold be amazing. It saves one mouse action.

**Comment by samhh** [comment]
There's already a shortcut that'll take you straight to the add dialog, check the README.

Tags is covered by #98.

Enter to save should work I think if the form is focused? Except for the tags where it'll add the tag instead.

**Comment by everyonus** [comment]
I agree, this extension needs a mode like that. Being able to view bookmarks from the browser is useful if you don't have lots of bookmark entries. If you do then (from my experience) it'll take awhile, possibly multiple minutes to load the extension.

> There's already a shortcut that'll take you straight to the add dialog, check the README.

While yes but it doesn't fill in the relevant details which makes it less useful then manually opening Bukubrow and clicking the "add a bookmark" button, as that automatically fills details
